src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc),
+    url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf);
+}
```

Comment:
You have the example for font-face in the checklist 
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/99403198/214285295-89a8b776-b070-466e-907f-e7bc40011e0f.png">